### PR TITLE
LTD-585: Parallel run setup and handle licence data extract type

### PR DESCRIPTION
**Pull request by saruniitr**
## Change description
Add new task for parallel run setup. This is to avoid confusion and to ensure we don't send messages to HMRC by mistake.
Add support for handling licence data extract.

### CODE REVIEW

File: mail/urls.py [hunk 1]
```diff
@@ -10,4 +10,5 @@
     # path("manage-inbox/", ManageInbox.as_view(), name="manage_inbox"),
     # path("send-usage-updates-to-lite-api/", SendUsageUpdatesToLiteApi.as_view(), name="send_updates_to_lite_api"),
     # path("send-licence-updates-to-hmrc/", SendLicenceUpdatesToHmrc.as_view(), name="send_updates_to_hmrc"),
+    path("parallel-run-outgoing/", TestParallelRunOutgoing.as_view(), name="parallel-run-outgoing"),
```
Comment (r4vi) [inline mail/urls.py:13]: maybe only enable the test endpoints `if DEBUG`?

Comment (saruniitr) [reply mail/urls.py:13]: :+1:

File: conf/settings.py [hunk 2]
```diff
@@ -103,6 +103,19 @@
 
 DATABASES = {"default": env.db()}
 
+
+INCOMING_EMAIL_PASSWORD = env("INCOMING_EMAIL_PASSWORD")
```
Comment (r4vi) [inline conf/settings.py:107]: this replaces the `EMAIL_PASSWORD` section?

can we update `local.env` example file?

File: mail/libraries/mailbox_service.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def read_last_three_emails(pop3connection: POP3_SSL) -> list:
     reversed_mails = list(reversed(mails))
     last_three_mails = reversed_mails[:3]
 
-    message_ids = [get_message_id(line) for line in last_three_mails]
+    message_ids = [get_message_id(line.decode("utf-8")) for line in last_three_mails]
```
Comment (r4vi) [inline mail/libraries/mailbox_service.py:38]: this might break tests

**Review by r4vi — COMMENTED** [review]
it would be good to remove the `parallel_run` name from functions and env vars as much as possible. 

left some small comments

File: conf/settings.py (hunk 2 above)
Comment (r4vi) [reply conf/settings.py:107]: if you set `local.env` with dummy values it will fix CI

File: mail/libraries/mailbox_service.py (hunk 3 above)
Comment (saruniitr) [reply mail/libraries/mailbox_service.py:38]: fixed the tests

**Review by r4vi — APPROVED** [review]

**Pull request merged**